cli/azd/pkg/environment/environment.go

```diff
@@ -36,6 +36,9 @@ const TenantIdEnvVarName = "AZURE_TENANT_ID"
 // to.
 const ContainerRegistryEndpointEnvVarName = "AZURE_CONTAINER_REGISTRY_ENDPOINT"
 
+// ContainerRegistryEndpointEnvVarName is the name of they key used to store the endpoint of the container registry to push to.
+const AksClusterEnvVarName = "AZURE_AKS_CLUSTER_NAME"
```

Comment:
Are we leading with support for a single cluster?  Might want to consider ways to support multiple clusters

Reply:
Agree that we need multi-cluster support or at least a way to deploy apps to different clusters.

Reply:
Going to start with single cluster for now.

Reply:
#1624